person.rb

```diff
@@ -0,0 +1,24 @@
+class Person
+  attr_accessor :name, :age, :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    true if age >= 18 && @parent_permission
+  end
```

Comment:
- [ ] `can_use_services` method should return true if person is of age **or** if they have permission from parents, however, your method returns true if person is of age **and** if they have permission from parents. Kindly fix your method to use `| |` instead of `&&`.

- [ ] Kindly make use of the `of_age?` method you have which returns `true` if `@age` is greater or equal to 18 and false otherwise to check if person is of age instead of rewriting the logic again. 

```suggestion
  def can_use_services?
    of_age? | | @parent_permission
  end
```